### PR TITLE
Well known topologies use case

### PR DESCRIPTION
**Pull request by EduPonz**

### CODE REVIEW

**Comment by EduPonz** [comment]
Test don't pass due to reference to sections in unmerged branches

File: docs/use-cases.rst [hunk 1]
```diff
@@ -26,14 +26,121 @@ Typical Use-Cases
...
+while at the same time limits the connections to those strictly necessary. As explained in the :ref:`discovery` section,
+all Fast-RTPS discovery mechanisms consist of two steps: Participant Discovery Phase (PDP), and Endpoint Discovery Phase
+(EDP).
+
+.. _wide_deployments_static_pdp:
+
+Peer-to-Peer Participant Discovery Phase
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, Fast-RTPS uses Simple Participant Discovery Protocol (SPDP) for PDP phase. This entails the participants
```
Comment (rsanchez15) [inline docs/use-cases.rst]: The long version of the acronym is no longer needed if you put it in the table.

File: docs/use-cases.rst [hunk 2]
```diff
@@ -26,14 +26,121 @@ Typical Use-Cases
...
+.. _wide_deployments_static:
+
+Well Known Network Topologies
+-----------------------------
+
+It is often the case in industrial deployments, such as productions lines, that the entire network topology (hosts, IP
+addresses, etc.) is known beforehand. Such scenarios are perfect candidates for Fast-RTPS STATIC discovery mechanism,
+which drastically reduces the middleware setup time (time until all the entities are ready for information exchange),
+while at the same time limits the connections to those strictly necessary. As explained in the :ref:`discovery` section,
+all Fast-RTPS discovery mechanisms consist of two steps: Participant Discovery Phase (PDP), and Endpoint Discovery Phase
```
Comment (rsanchez15) [inline docs/use-cases.rst]: You state the long version of the acronyms in the introduction. Maybe extend them here could be repetitive.

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/176/)

**Pull request merged**